source/adapters/level_zero/CMakeLists.txt

```diff
@@ -33,8 +33,12 @@ if (NOT DEFINED LEVEL_ZERO_LIBRARY OR NOT DEFINED LEVEL_ZERO_INCLUDE_DIR)
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unknown-warning-option")
     endif()
 
-    set(LEVEL_ZERO_LOADER_REPO "https://github.com/oneapi-src/level-zero.git")
-    set(LEVEL_ZERO_LOADER_TAG v1.16.1)
+    if (NOT DEFINED LEVEL_ZERO_LOADER_REPO)
+        set(LEVEL_ZERO_LOADER_REPO "https://github.com/oneapi-src/level-zero.git")
+    endif()
+    if (NOT DEFINED LEVEL_ZERO_LOADER_TAG)
+        set(LEVEL_ZERO_LOADER_TAG v1.16.1)
+    endif()
```

Comment:
Are these also intended to be overridden at CMake configure time?

Reply:
Yes, some projects use github mirrors, so there is a need for them to replace this repo.
Changed this per suggestion above as well.